script-engine/src/main/java/org/ihtsdo/termserver/scripting/fixes/BatchFix.java

```diff
@@ -297,6 +297,7 @@ private void createTask(Task task) throws TermServerScriptException, Interrupted
 					task.setKey(scaClient.createTask(project.getKey(), task.getSummary(), taskDescription));
 					debug ("Creating task branch in terminology server: " + task);
 					task.setBranchPath(tsClient.createBranch(project.getBranchPath(), task.getKey()));
+					tsClient.setAuthorFlag(task.getBranchPath(), "batch-change", "true");
```

Comment:
After creating a task on AP/Jira, the script will then send a request to Snowstorm to set the appropriate flag.